site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -83,17 +83,12 @@ To set a logpoint:
 1.  Go to the line of code.
 1.  To the left of the line of code is the line number column. Right-click it.
 1.  Select **Add logpoint**. A dialog displays underneath the line of code.
-1.  Enter your log message in the dialog.
-    
-    You can separate strings, variable, and objects with commas. Wrap strings in quotes (`" "`), objects in curly brackets (`{ }`), and type variable names as they are.
-
-    For example: `"I'm a string.", variableName, {object}`.
-
+1.  Enter your log message in the dialog. You can use the same syntax as you would with a [`console.log(message)`](https://developer.mozilla.org/docs/Web/API/Console/log) call.
```

Comment:
perhaps, point to our documentation of `console.log` instead of MDN.

Reply:
I think the the example prior to this change is helpful. I suggest instead of removing them all, rephrase them properly and use it here (e.g. `"a string", object, variable, variable.length > 1`).

Reply:
Our console.log doc lacks syntax explanation: https://developer.chrome.com/docs/devtools/console/api/#log. Something to fix later.